tests/service/data/SyntaxTree/Attribute/RangeOfAttribute.fs

```diff
@@ -1,3 +1,3 @@
 
-[<MyAttribute(foo ="bar")>]
+[<MyAttribute(foo = "bar")>]
```

Comment:
Is there a reason for this change, apart from the style changes?

Reply:
The style change I think is coincidental, I was playing with that test example to figure out how the that section of the testing framework was (deleting the arg to see the .actual, etc) and typed the arg back in from memory when I should have reverted the file. I'll revert that in the next commit.